front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -551,19 +555,28 @@ export default function AssistantBuilder({
               onSave={
                 submitEnabled
                   ? () => {
+                      setIsSavingOrDeleting(true);
                       submitForm()
-                        .then(() => {
+                        .then(async () => {
+                          await mutate(
+                            `/api/w/${owner.sId}/assistant/agent_configurations`
+                          );
                           void router.push(
                             `/w/${owner.sId}/builder/assistants`
                           );
+                          setIsSavingOrDeleting(false);
```

Comment:
Do we want to do this in a "finally" ? or at least also do it in case of error ?

Reply:
My bad fixing

Reply:
added to error case :+1: